src/app/video-pair/video-pair.component.css

```diff
@@ -0,0 +1,8 @@
+video-pair {
```

Comment:
This isn't `less`, and it's not immediately obvious from the running app if it's working or not. I suspect not. Investigate... at least a little bit.

Reply:
It was not. It is now.